examples/benchmarks/bench_cg.py

```diff
@@ -137,11 +138,11 @@ def timeit(calc, iters, args):
 
 def test():
     print("\nTesting benchmark functions...")
-    A, b, x0 = setup_input(50)  # dense A
+    A, b, x0 = setup_input(50, 7)  # dense A
```

Comment:
If you are going to add a new param explicitly, can you change this to be named parameters ? i.e. `setup_input(n=50, sparsity=7)`

Reply:
Done